README.md

```diff
@@ -98,10 +98,54 @@ print(json.dumps(result_sar, sort_keys=True, indent=4, separators=(',', ': ')))
 
 Returns a SeDAS search result object. [See more](https://geobrowser.satapps.org/docs/json_SearchResponse.html)
 
-### Filtering on a group of satellites
+### List the source groups available to the user
 
-Possible filter groups for all users are: 'S1' and 'AIRSAR'.
-Additionally, if you have been approved for SSGP access you can filter on: 'Cosmo-SkyMed', 'SPOT' and 'Pleiades'.
+```python
+import json
+from sedas_pyapi.sedas_api import SeDASAPI
+from getpass import getpass
+
+_username = input("Please enter your username:")

```

Comment:
though I have just got rid of the underscores here in the last pr

Reply:
Rats, I thought I'd caught them all.